rules/terraform_locals_order.go

```diff
@@ -0,0 +1,124 @@
+package rules
+
+import (
+	"fmt"
+	"github.com/hashicorp/hcl/v2"
+	"sort"
+	"strings"
+
+	"github.com/hashicorp/hcl/v2/hclsyntax"
+	"github.com/hashicorp/hcl/v2/hclwrite"
+	"github.com/terraform-linters/tflint-plugin-sdk/tflint"
+	"github.com/terraform-linters/tflint-ruleset-terraform/project"
+)
+
+// TerraformLocalsOrderRule checks whether comments use the preferred syntax
+type TerraformLocalsOrderRule struct {
+	tflint.DefaultRule
+}
+
+// NewTerraformLocalsOrderRule returns a new rule
+func NewTerraformLocalsOrderRule() *TerraformLocalsOrderRule {
+	return &TerraformLocalsOrderRule{}
+}
+
+// Name returns the rule name
+func (r *TerraformLocalsOrderRule) Name() string {
+	return "terraform_locals_order"
+}
+
+// Enabled returns whether the rule is enabled by default
+func (r *TerraformLocalsOrderRule) Enabled() bool {
+	return false
+}
+
+// Severity returns the rule severity
+func (r *TerraformLocalsOrderRule) Severity() tflint.Severity {
+	return tflint.NOTICE
+}
+
+// Link returns the rule reference link
+func (r *TerraformLocalsOrderRule) Link() string {
+	return project.ReferenceLink(r.Name())
+}
+
+// Check checks whether single line comments is used
+func (r *TerraformLocalsOrderRule) Check(runner tflint.Runner) error {
+	files, err := runner.GetFiles()
+	if err != nil {
+		return err
+	}
+	for _, file := range files {
+		if err = r.checkFile(runner, file); err != nil {
+			return err
+		}
+	}
+	return nil
+}
+
+func (r *TerraformLocalsOrderRule) checkFile(runner tflint.Runner, file *hcl.File) error {
+	blocks := file.Body.(*hclsyntax.Body).Blocks
```

Comment:
This causes a panic in JSON syntax.
https://www.terraform.io/language/syntax/json